bitnami/dataplatform-bp1/values.yaml

```diff
@@ -0,0 +1,233 @@
+##
+spark:
+  enabled: true
+  ## Spark master specific configuration
+  ##
+  master:
+    resources:
+      # We usually recommend not to specify default resources and to leave this as a conscious
+      # choice for the user. This also increases chances charts run on environments with little
+      # resources, such as Minikube. If you do want to specify resources, uncomment the following
+      # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      limits: {}
+      #  cpu: 250m
+      #  memory: 1Gi
+      requests:
+        cpu: 250m
+        memory: 1Gi
+    affinity:    
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: spark
+              app.kubernetes.io/instance: dp-spark
+              app.kubernetes.io/component: worker
+          topologyKey: "kubernetes.io/hostname"
+  ## Spark worker specific configuration
+  ##
+  worker:
+    replicaCount: 2
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: spark
+              app.kubernetes.io/instance: dp-spark
+              app.kubernetes.io/component: worker
+          topologyKey: "kubernetes.io/hostname"
+    resources:
+      # We usually recommend not to specify default resources and to leave this as a conscious
+      # choice for the user. This also increases chances charts run on environments with little
+      # resources, such as Minikube. If you do want to specify resources, uncomment the following
+      # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      limits: {}
+      #  cpu: 250m
+      #  memory: 1Gi
+      requests: 
+        cpu: 250m
+        memory: 1Gi
+
+## Solr Subchart parameters
+##
+solr:
+  enabled: true
+  replicaCount: 2
+  affinity: 
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/name
+            operator: In
+            values:
+            - solr
+        topologyKey: "kubernetes.io/hostname"   
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {}
+    #  cpu: 1000m
+    #  memory: 1Gi
+    requests:
+      cpu: 250m
+      memory: 1Gi
+  zookeeper:
+    enabled: false
+  ## This value is only used when zookeeper.enabled is set to false
+  ##
+  externalZookeeper:
+    ## Server or list of external zookeeper servers to use.
+    ##
+    servers: []
```

Comment:
Don't we want to connect Solr with Zookeeper?
Also, we should wait until we finish the Solr chart to configure this one, because I guess we would like to enable the Cloud mode for Solr, so the parameters for that should also appear in the Solr section.

Reply:
Yes, I just kept it blank for now, was waiting for solr cloud image.